content/en/docs/kubernetes/getting-started.md

```diff
@@ -0,0 +1,276 @@
+---
+title: Getting Started
+weight: 1
+# prettier-ignore
+cSpell:ignore: kubeletstats kubelet filelog sattributes sattributesprocessor kubeletstatsreceiver filelogreceiver loggingexporter otlpexporter sclusterreceiver sobjectsreceiver
+---
+
+This section will walk you through the fasted way to get started monitoring your
+Kubernetes cluster using OpenTelemetry. It will focus on collecting metrics and
+logs for Kubernetes clusters, nodes, pods, and containers as well as enabling
+the cluster to support services emitting OTLP data.
+
+If you're looking to see OpenTelemetry in action in Kubernetes the best place to
+start is the [OpenTelemetry Demo](../../demo/kubernetes-deployment). The demo is
+intended to illustrate the implementation of OpenTelemetry, but it is not
+intended to be an example of how to monitor Kubernetes itself. Once you're done
+with this walkthrough it can be a fun experiment to install the demo and see how
+all all the monitoring responds to an active workload.
+
+If you're looking to start migrating from Prometheus to OpenTelemetry, or if
+you're interested in using the OpenTelemetry Collector to collect Prometheus
+metrics, see [Prometheus Receiver](../collector/components#prometheus-receiver).
+
+## Overview
+
+Kubernetes exposes a lot of important telemetry in many different ways. It has
+logs, events, metrics for many different objects, and the data generated by its
+workloads.
+
+To collect all of this data we'll use the
+[OpenTelemetry Collector](../../collector/). The collector has many different
+tools at its disposal which allow it to efficiently collect all this data and
+enhance it in meaningful ways.
+
+To collect all the data we'll need two installations of the collector, one as a
+[Daemonset](../../collector/deployment/agent) and one as a
+[Deployment](../../collector/deployment/gateway). The Daemonset installation of
+the collector will be used to collector telemetry emitted by services, logs, and
+metrics for nodes, pods, and containers. The deployment installation of the
+collector will be used to collect metrics for the cluster and events.
+
+To install the collector we'll use the
+[OpenTelemetry Collector Helm chart](../helm/collector), which comes with a few
+configuration options that will make configure the collector easier. If you're
+unfamiliar with Helm, checkout [Helm](https://helm.sh/). If you're interested in
+using a Kubernetes operator, see [OpenTelemetry Operator](../operator/), but
+this guide will focus on the Helm chart.
+
+## Preparation
+
+This guide will assume the use of a [Kind cluster](https://kind.sigs.k8s.io/),
+but you're free to use any Kubernetes cluster that you deem fit.
+
+Assuming you already have
+[Kind installed](https://kind.sigs.k8s.io/#installation-and-usage), create a new
+kind cluster:
+
+```sh
+kind create cluster
+```
+
+Assuming you already have [Helm installed](https://helm.sh/docs/intro/install/),
```

Comment:
@TylerHelmuth Is this a 2nd step or is this an alternative path in the installation process? It may be helpful to add numbering to these steps.

Reply:
They can be done in any order.